src/Poster/Poster.stories.js

```diff
@@ -0,0 +1,230 @@
+  </Poster>
+);
+
+const PosterTemplateImageLeft = ({ ...rest }) => (
+  <Poster {...rest}>
+    <Poster.Image className="su-rs-mr-4">
+      <img
+        src="https://placekitten.com/600/400"
+        className="su-object-cover su-w-full su-h-full su-object-center"
+        alt=""
+      />
+    </Poster.Image>
+
+    <Poster.Content className="su-items-start md:su-flex-grow">
+      <Poster.Headline>Title</Poster.Headline>
+      <p className="su-big-paragraph su-leading-snug">
+        Cras varius. Ut non enim eleifend felis pretium feugiat. Suspendisse
+        feugiat. Maecenas egestas arcu quis ligula mattis placerat. Suspendisse
+        faucibus, nunc et pellentesque egestas, lacus ante convallis tellus,
+        vitae iaculis lacus elit id tortor.
+      </p>
+    </Poster.Content>
+  </Poster>
+);
+
+const PosterTemplateCustom = ({ ...rest }) => (
+  <Poster {...rest}>
+    <Poster.Image className="su-rs-mb-2">
+      <img
+        src="https://placekitten.com/600/400"
+        className="su-object-cover su-w-full su-h-full su-object-center"
+        alt=""
+      />
+    </Poster.Image>
+    <Poster.Content className="su-text-white su-items-center su-text-center su-align-center">
+      <Poster.Headline>Title</Poster.Headline>
+      <p className="su-big-paragraph su-leading-snug">
+        Cras varius. Ut non enim eleifend felis pretium feugiat. Suspendisse
+        feugiat. Maecenas egestas arcu quis ligula mattis placerat. Suspendisse
+        faucibus, nunc et pellentesque egestas, lacus ante convallis tellus,
+        vitae iaculis lacus elit id tortor.
+      </p>
+    </Poster.Content>
+  </Poster>
+);
+
+const PosterTemplateCustomLeft = ({ ...rest }) => (
+  <Poster {...rest}>
+    <Poster.Image className="su-rs-mr-4">
+      <img
+        src="https://placekitten.com/600/400"
+        className="su-object-cover su-w-full su-h-full su-object-center"
+        alt=""
+      />
+    </Poster.Image>
+    <Poster.Content className="su-text-white su-items-start md:su-flex-grow">
+      <Poster.Headline>Title</Poster.Headline>
+      <p className="su-big-paragraph su-leading-snug">
+        Cras varius. Ut non enim eleifend felis pretium feugiat. Suspendisse
+        feugiat. Maecenas egestas arcu quis ligula mattis placerat. Suspendisse
+        faucibus, nunc et pellentesque egestas, lacus ante convallis tellus,
+        vitae iaculis lacus elit id tortor.
+      </p>
+      <CtaButton
+        text="Learn more"
+        srText="about Decanter"
+        animate="right"
+        icon="more"
+        variant="unset"
+        size="unset"
+        href="https://decanter.stanford.edu"
+        className="su-rs-mt-0 su-bg-white hocus:su-bg-lagunita su-text-black hocus:su-text-white su-px-20 su-pt-14 su-pb-16"
+      />
+    </Poster.Content>
+  </Poster>
+);
+
+const PosterTemplateCustomBorder = ({ ...rest }) => (
+  <Poster {...rest}>
+    <Poster.Image className="su-border-3 su-border-solid su-border-digital-red-light su-rs-mb-2">
```

Comment:
Looks like Figma image border is a 7px one, but Decanter v7 doesn't have a 7px width currently. I know @sherakama added some recently, so let's go with either 6px or 8px 😄  Although... @sherakama , is there a release that got punched with your border width PR?
https://github.com/SU-SWS/decanter/pull/834/files

Reply:
The latest release does not seem to include the border-width 😢  Currently the available options are
```
.su-border-0{
  border-width: 0
}

.su-border-2{
  border-width: 2px
}

.su-border-3{
  border-width: 3px
}

.su-border{
  border-width: 1px
}
```

Reply:
Send in a PR to add border-7 and add the `alpha` label to it. That should get you what you need.